src/Glpi/Form/Export/Serializer/FormSerializer.php

```diff
@@ -125,6 +172,11 @@ public function importFormsFromJson(
             $requirements = $form_spec->data_requirements;
             $mapper->mapExistingItemsForRequirements($requirements);
 
+            $form_name = $form_spec->name;
+            if (in_array($form_name, $skipped_forms)) {
+                continue;
+            }
+
```

Comment:
I think you forgot to switch to IDs here ;).
(It also mean the test suite is incomplete as it should catch this error).